libwallet/src/migration.rs

```diff
@@ -0,0 +1,254 @@
+// Copyright 2022 The Epic Developers
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Add a description for this file

Reply:
Can you give me an example?

Reply:
For sure!
This file, for example, has some descriptions on what is its objective:
https://github.com/EpicCash/epic-wallet/blob/66a34b49af1af624f6a5b55d8bdcd6eab8524dbc/api/src/lib.rs#L15-L17

This one also has a good example (Although, this should be before the imports):
https://github.com/EpicCash/epic-wallet/blob/66a34b49af1af624f6a5b55d8bdcd6eab8524dbc/api/src/owner.rs#L42-L54

Reply:
Thanks, Got it, I opened 2 or 3 files yesterday trying to look for some description in the header and I didn't find it.